querybook/webapp/components/TopTierCrown/TopTierCrown.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import './TopTierCrown.scss';
+import { Icon } from 'ui/Icon/Icon';
+
+export const TopTierCrown: React.FunctionComponent<{
+    showTooltip?: boolean;
+    tooltipPos?: string;
+}> = ({ showTooltip = false, tooltipPos = 'down' }) => {
+    if (showTooltip) {
+        return (
+            <span
+                className="TopTierCrown flex-row"
+                aria-label="Top Tier"
+                data-balloon-pos={tooltipPos}
+            >
+                <Icon className="ml4" name="Crown" />
```

Comment:
ml4 should be outside

Reply:
also i think its better to just pass a className, since hard coding ml4 would be depending on the outer context?